common/types/fixationEntry.go

```diff
@@ -0,0 +1,16 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// IsEntryStale tests whether an entry is stale, i.e. has refcount zero _and_
+// has passed its stale_at time (more than STALE_ENTRY_TIME since deletion).
+func (entry Entry) IsStale(ctx sdk.Context) bool {
+	if entry.GetRefcount() == 0 {
+		if entry.StaleAt < uint64(ctx.BlockHeight()) {
```

Comment:
no need for nesting, can use &&

Reply:
Yes. This form intended for the reader - emphasize the two tests.